cloudinit/cmd/main.py

```diff
@@ -328,9 +328,12 @@ def main_init(name, args):
     outfmt = None
     errfmt = None
     try:
-        early_logs.append((logging.DEBUG, "Closing stdin."))
-        util.close_stdin()
-        (outfmt, errfmt) = util.fixup_output(init.cfg, name)
+        if not os.isatty(sys.stdin.fileno()):
```

Comment:
Nit: These 5 lines are duplicated a few other times. Any reason we can't move them all into `util.close_stdin()` so each call site is just the single call to the function?

Reply:
I also bumped this function over to `cloudinit/cmd/main.py` since it isn't used anywhere else